npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -203,14 +205,23 @@ export async function getAngularJson (projectRoot: string): Promise<AngularJson>
   return JSON.parse(angularJson)
 }
 
+function buildLogger (name: string = ''): logging.Logger {
```

Comment:
Seems this is only called once (below) and `name` is never passed, is that required, or can we remove the argument entirely?

Reply:
It's actually called in two places, one of which is within itself so it's easy to miss. We call it once with no name for the root logger, and the `createChild` signature accepts a name for child loggers